### DISCUSSION TITLE
[960990] Beyond: Two Souls | 本体 +dlc

### DISCUSSION THREAD

**Discussion by lanlan598215** [pjy612/SteamManifestCache#58] [Share]
appid:960990
Keys:
960991;120e868e509a7eb6f5fcf3924ff7726c5db98bceb8340ca8e16aa373acc0fe5c
960992;dede946a447b25c049863aa0ced1b09195c52e5377a8022e12e4ca4509080ee9
960993;7c073d979171b26c99ee683e9918c558408b6144231cea9341ed09789459bc7c
960994;ee093b5e12f97397950fb7af50394e1d9c256868d2d1edd40a135205dd714167
960995;f22b1bc4870e917078a956d1efd7dee80d92c9616e7dadc8b60e61a09464e4a9
960996;d7e856a761b4c499a11e8a8fd695e938d4fd8be141360bfe5d0e6b2d1e6a2ce0
960997;4e53411c0ef8214d273205bf92c70ef42bf786b70ccc1e1acfbf08ec2630b077
960998;0366b83db473e8e4c0f960a151fe9391143ab7d224de1a894c7202e63be0c7d7
960999;444f4fb2ad630406d92c7056bd7baa3fd5f5f882091f5778fdd64edd24108846
code::8D63D0FFAA30E612
email:[1141167182@qq.com]
[Beyond Two Souls.zip](https://github.com/pjy612/SteamManifestCache/files/13693116/Beyond.Two.Souls.zip)

**Comment by ZEERDEER** [comment]
请文件上传完成后再提交评论

**Comment by pjy612** [comment]
请编辑下 至少确保上传完成。
从 uploading 变成别的链接后 才表示上传成功。

**Reply by lanlan598215** [reply]
好的，我弄下

**Reply by pjy612** [reply]
。。。你直接编辑原贴重新上传不就好了。。。帮你处理下。。。

ps。查收

**Reply by lanlan598215** [reply]
我刚刚用这个，不好意思，另外那个发来的怎么使用

**Reply by pjy612** [reply]
解压 扔 SAI 旁边，一般来说 再次使用SAI能看出不同

**Reply by lanlan598215** [reply]
好的，正在使用
